ruby-challenges.rb

```diff
@@ -14,18 +21,32 @@
 
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
+
+
 album1 = 'Rubber Soul'
+puts album1.delete "a,e,i,o,u"
+
+
 # Expected output: 'Rbbr Sl'
 album2 = 'Sgt Pepper'
+puts album2.delete "a,e,i,o,u"
 # Expected output: 'Sgt Pppr'
 album3 = 'Abbey Road'
+puts album3.delete "a,e,i,o,u"
 # Expected output: 'bby Rd'
 
 
 
 # -------------------3) Create a method that takes in a string and checks if the string is a palindrome. A palindrome is the same word spelled forwards or backwards. Use the test variables provided.
 
 is_palindrome1 = 'Racecar'
+
+def palindrome (string)
+    if string = string.reverse
```

Comment:
Check the output in your terminal. This code will not give the expected out for Racecar. You will need to take in consider if letters are lower or upper case.